modules/cnf-configuring-node-groups-for-the-numaresourcesoperator.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-numa-aware-scheduling.adoc
+
+:_content-type: PROCEDURE
+
+[id="cnf-configuring-node-groups-for-the-numaresourcesoperator_{context}"]
+= Configuring NUMA Resource Operator polling
+
+You can configure how the NUMA Resources Operator polls a `nodeGroup` for available NUMA resources by configuing the `spec.nodeGroups` specification in the `NUMAResourcesOperator` custom resource (CR). The configuration options are the following:
+
+* `infoRefreshMode`: Determines the trigger for polling updates to the API Server.
+* `infoRefreshPeriod`: Determines the duration between polling updates.
+* `podsFingerprinting`: Determines if point-in-time information for the current set of pods running on a node is exposed in polling updates.
```

Comment:
I am a little unclear about what information is gathered in pod finger printing. Perhaps need to some help adding more detail?